### DISCUSSION TITLE
Comparing regions

### DISCUSSION THREAD

**Discussion by jesuisalexjamet** [electricitymaps/electricitymaps-contrib#6048] [Ideas]
**What is the potential new feature all about?**
Browsing on X (ex-Twitter), I tend to see many posts drawing comparisons between two countries (I'm French, so if we can take the chance to make fun of Germans, we do 😄). But I believe that to be a reasonably good proposal for a new feature on the app.

**What would that look like?**
See a comparison website for phones or any comparable goods? That. We could have a tool allowing the user to select 2+ countries (or regions), and the app would show a comparison in terms of emissions per unit of energy consumed (or any other metric).

There is the idea! Let's see what the community thinks. Cheers! 😄

**Comment by VIKTORVAV99** [comment]
Since this is not really an issue but rather a feature request I will turn it into a discussion instead as it is better suited for this.

With that said I do think it would be nice to have something like this in place, it would take some time to develop and is not a priority at the moment but we'll keep it in mind!

**Reply by jesuisalexjamet** [reply]
Hello! Thanks for considering the idea. I do understand that it is probably far from a trivial implementation but if that can help people to make sense of the metrics, I think it could be a great addition.

Thanks again! 😄
